org.mixedrealitytoolkit.uxcore/Slider/SliderSounds.cs

```diff
@@ -171,45 +152,51 @@ private void Start()
             slider.OnValueUpdated.AddListener(OnValueUpdated);
         }
 
+        private static float NeutralPitchShift = 1f;
+        private float passNotchPitchShift = NeutralPitchShift;
+ 
         private void OnValueUpdated(SliderEventData eventData)
         {
-            if (!(playSoundsOnlyOnInteract && !isInteracting) && playTickSounds && passNotchAudioSource != null && passNotchSound != null)
+
+            if (!(playSoundsOnlyOnInteract && !isInteracting) && playTickSounds && AudioSource != null && passNotchSound != null)
             {
+                AudioSource.pitch = passNotchPitchShift;
```

Comment:
Since AudioSource.pitch is recalculated every time (line 169) before playing the passNotchSound, what is the purpose of storing the passNotchPitchShift and setting it here?

Reply:
i was being cautious :smile:

i can remove